### PR TITLE
NO-TICKET - Changed algorithm for sending transactions in fee relayer…

### PR DESCRIPTION
**Pull request by 1sakov**

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/feerelayer/FeeRelayerModule.kt [hunk 1]
```diff
@@ -50,7 +50,7 @@ object FeeRelayerModule : InjectionModule {
             )
         }
 
-        factory { FeeRelayerInteractor(get(), get(), get(), get(), get(), get(), get()) }
+        factory { FeeRelayerInteractor(get(), get(), get(), get(), get(), get(), get(), get()) }
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/feerelayer/FeeRelayerModule.kt:53]: factoryOf?

File: app/src/main/java/org/p2p/wallet/feerelayer/interactor/FeeRelayerInteractor.kt [hunk 2]
```diff
@@ -223,13 +227,10 @@ class FeeRelayerInteractor(
...
         transaction.sign(preparedTransaction.signers)
 
-        /*
-        * Retrying 3 times to avoid some errors
-        * For example: fee relayer balance is not updated yet and request will fail with insufficient balance error
-        * */
-        return retryRequest {
-            feeRelayerRepository.relayTransaction(transaction, statistics)
-        }
+        val serializedTransaction = Base64Utils.encode(transaction.serialize()).toBase64Instance()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/feerelayer/interactor/FeeRelayerInteractor.kt:230]: Base64Utils.encode(transaction.serialize()).toBase64Instance() -> transaction.serialize().toBase64Instance()

File: app/src/main/java/org/p2p/wallet/rpc/interactor/TransactionInteractor.kt [hunk 3]
```diff
@@ -61,11 +62,11 @@ class TransactionInteractor(
         }
     }
 
-    suspend fun sendTransaction(signedTransaction: String, isSimulation: Boolean): String {
+    suspend fun sendTransaction(signedTransaction: Base64String, isSimulation: Boolean): String {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/rpc/interactor/TransactionInteractor.kt:65]: thanks!

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt [hunk 4]
```diff
@@ -74,7 +74,7 @@ class BridgeSendInteractor(
             currency = token.mintAddress
         )
         return feeRelayerRepository.signTransaction(
-            transaction = signedTransaction.transaction.base58Value.toBase64String(),
+            transaction = signedTransaction.transaction.base58Value.toBase64Instance(),
```
Comment (NikolaiEmelianov) [inline app/src/main/java/org/p2p/wallet/bridge/send/BridgeSendInteractor.kt:77]: i change it to `signedTransaction.transaction.decodeToBytes().toBase64Instance()`

**Review by NikolaiEmelianov — APPROVED** [review]

**Pull request merged**